R/run-log.R

```diff
@@ -42,16 +42,23 @@ run_log <- function(.base_dir, .recurse = TRUE) {
 #' and attempts to read them to a model object with [safe_read_model()].
 #' @param .base_dir Directory to search for model YAML files.
 #' @param .recurse If `TRUE` search recursively in subdirectories as well.
+#' @param .exclude Provides filter for runs based on an input vector.
 #' @importFrom stringr str_subset
 #' @importFrom purrr map_lgl map compact
 #' @importFrom fs dir_ls
 #' @keywords internal
-find_models <- function(.base_dir, .recurse) {
+find_models <- function(.base_dir, .recurse , .exclude = NULL) {
```

Comment:
`run_log`, `config_log`, and `summary_log` use `.exclude = vector()`.  Any reason to go with NULL here?   For consistency, I think it'd be good to have this be the same.  Or, given this is an internal function and all callers pass `.exclude`, I also think it'd be fine to just drop the default value from `find_models`.